### ISSUE TITLE
PvP vendor in Gadgetzan not selling tbc season 4 equipment.

### ISSUE THREAD

**Issue by Nailjam** [SunwellTracker/issues#3103]
Decription: In Gadgetzan there are PvP vendors that should sell the old level 70 pvp gear.

How it works: Right now the vendors sell level 80 PvP gear.

How it should work: There should be vendors there that sell you the old PvP equipment.

Source (you should point out proofs of your report, please give us some source): https://www.wowhead.com/item-set=-1/brutal-gladiators-aegis#comments:id=736475
The comment made by "Kaleanon" confirms the vendors were still there when he made the comment in patch 3.1.3 I have no proof that they were still there in 3.3.5, but there is also no proof they removed it.

**Comment by Nailjam** [comment]
Cant figure out how to link to the correct comment, the one I'm refering to is lower down. 

> Currently All Season 4 Arena gear can still be Picked up from the Brutal Gladiator Vendors Located in Dalaran and Gadgetzan. (Not sure why but the vendor in Shattrath was taken out last time I checked)

There have no rating required on any piece and are sold for 1150-1500 a piece. Only requirement is they are arena point costs and level 70.

For the Bored, the Role player or those who just wanted to have some really great looking sets (ones that look almost exactuly like gear from the Sunwell raid) Then this is a great chance to pick them up without having to raid and roll on gear.

So in short,

1. Yes this Gear is No rating required
2. Purchasible From the Brutal Gladiator Vendor in Gadgetzan or Dalaran
3. Still looks amazing.>

**Comment by nueschts** [comment]
The vendor is in fact in the game, just not where you would expect :)

https://imgur.com/a/YFduTeN
